packages/unmock-node/src/__tests__/with-credentials.test.ts

```diff
@@ -30,4 +29,11 @@ test("credentials written to .unmock/credentials work just like a token", async
     "https://www.behance.net/v2/projects?api_key=u_n_m_o_c_k_200",
   );
   expect(typeof projects[0].id).toBe("number");
+  // test to make sure cache works
+  const {
+    data,
+  } = await axios(
+    "https://www.behance.net/v2/projects?api_key=u_n_m_o_c_k_200",
+  );
+  expect(typeof data.projects[0].id).toBe("number");
```

Comment:
Is this a visual test (based on the logger output)?
If so, it might be better to provide a `MockLogger` class so we can check if it used "cached" or not :thinking:

Reply:
True.